libs/utils/android/system.py

```diff
@@ -153,6 +153,26 @@ def start_action(target, action, action_args=''):
         target.execute('am start -a {} {}'.format(action, action_args))
 
     @staticmethod
+    def view_uri(target, uri, force_new=True):
+        """
+        Start a view activity by specifying in URI
+
+        :param uri: URI of the item to display
+        :type uri: str
+
+        :param force_new: Force the viewing application to be
+            relaunched if it is already running
+        :type force_new: bool
+        """
+        arguments = '-d {}'.format(uri)
+
+        if force_new:
+            # Flag is ACTIVITY_NEW_TASK & ACTIVITY_CLEAR_TASK
+            arguments = '{} -f 0x10008000'.format(arguments)
```

Comment:
Can we defined this ugly hex as a constant value with a meaningful name at the top of the class?